docker/static/Dockerfile

```
@@ -12,7 +13,8 @@ RUN mkdir -p \
    curl -sL https://download.oracle.com/otn_software/java/jdk/17.0.1+12/2a2082e5a09d4267845be086888add4f/jdk-17.0.1_doc-all.zip --output /tmp/jdk.zip && \
       unzip -q -d /tmp/ /tmp/jdk.zip && \
       mv /tmp/docs/* /usr/share/nginx/html/docs/java/en/ && \
-      rm -rf /tmp/jdk.zip /tmp/docs
+      rm -rf /tmp/jdk.zip /tmp/docs \ &&
```

welp esto estaba al revés:

```suggestion
      rm -rf /tmp/jdk.zip /tmp/docs && \
```

el `\` al final de una línea le indica a Docker que el comando continúa en la línea siguiente. ya está corregido en `main`.